frostsnap_coordinator/src/backup_run.rs

```diff
@@ -1,43 +1,40 @@
 use crate::persist::{Persist, ToStringWrapper};
 use bdk_chain::rusqlite_impl::migrate_schema;
-use frostsnap_core::{AccessStructureId, AccessStructureRef, DeviceId, Gist, KeyId};
+use frostsnap_core::{AccessStructureId, AccessStructureRef, Gist, KeyId};
 use rusqlite::params;
 use std::collections::BTreeMap;
-use std::time::{SystemTime, UNIX_EPOCH};
 use tracing::{event, Level};
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct DisplayBackupState {
+    pub confirmed: bool,
+    pub legacy_display_confirmed: bool,
```

Comment:
Why do we need this `legacy_display_confirmed` field?

Can't we just map legacy `DisplayBackupConfirmed` messages to `confirmed: true`? The share is backed up either way.

The current code skips `markBackupComplete` for legacy devices:
```dart
if (isComplete && !result.legacyDisplayConfirmed) {
  await coord.markBackupComplete(...);
}
```

But legacy devices are still backing up a specific share at a specific index. Then we wouldn't need a new `BackupRecorded` enum variant either. Is there a reason we need to distinguish between legacy and new confirmations?